### PR TITLE
Bulk Group and Folder Permissions Endpoints

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: src/iplant_groups/routes/folders.clj [hunk 1]
```diff
@@ -8,69 +8,80 @@
 
 (defroutes folders
   (GET "/" []
-        :query       [params SearchParams]
-        :return      FolderList
-        :summary     "Folder Search"
-        :description "This endpoint allows callers to search for folders by name. Only folders
-        that are visible to the given user will be listed."
-        (ok (folders/folder-search params)))
```
Comment (slr71) [inline src/iplant_groups/routes/folders.clj:16]: This was a minor change, but I think the reduced indentation helps with nested, contexts.

File: src/iplant_groups/routes/schemas/privileges.clj [hunk 2]
```diff
@@ -5,9 +5,23 @@
             [iplant_groups.routes.schemas.folder :as folder]
             [schema.core :as s]))
 
-(def ValidFolderPrivileges (s/enum "create" "stem" "stemAdmin" "stemAttrRead" "stemAttrUpdate"))
```
Comment (slr71) [inline src/iplant_groups/routes/schemas/privileges.clj:8]: Grouper was complaining about `stemAdmin` when I was using it for testing, so I removed it from the set of options.

File: src/iplant_groups/clients/grouper.clj [hunk 3]
```diff
@@ -500,26 +506,26 @@
...
-  [entity-lookup allowed? username subject-id privilege-names]
+  [entity-lookup allowed? username subject-ids privilege-names]
   {:WsRestAssignGrouperPrivilegesRequest
    (assoc entity-lookup
      :actAsSubjectLookup (act-as-subject-lookup username)
      :clientVersion "v2_2_000"
      :privilegeNames privilege-names
      :allowed (if allowed? "T" "F")
-     :wsSubjectLookups [{:subjectId subject-id}])})
+     :wsSubjectLookups (vec (for [subject-id subject-ids] {:subjectId subject-id})))})
```
Comment (psarando) [inline src/iplant_groups/clients/grouper.clj:516]: 👍

**Review by psarando — APPROVED** [review]
LGTM 👍

**Comment by slr71** [comment]
Thanks for the review. 👍

**Pull request merged**